src/slider/slider.scss

```diff
@@ -0,0 +1,301 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+$thumb-height: $iui-m;  // 16px
```

Comment:
Not sure about variables which refer just a single other variable...
Could have used `$iui-m` just directly.. Unless it gives really a better understanding of the code..?

Reply:
I like it because is make it clear when I use something like, $thumb-height * 0.5, exactly what I am trying to achieve. If we decide to make $thumb-height = $iui-m + $iui-xxs; nothing else has to change.